### DISCUSSION TITLE
为什么只要一点击下载就停止工作？

### DISCUSSION THREAD

**Discussion by dididididxy** [nilaoda/N_m3u8DL-CLI#487] [General]
![IMG20210808164827](https://user-images.githubusercontent.com/88639744/128654151-31f3394a-b66a-4d17-9c40-f914c18c4a37.jpg)
如图

**Comment by nilaoda** [comment]
你的系统是不是精简过。。

**Reply by dididididxy** [reply]
我不知道，如果是系统精简过有办法解决使用吗？如果没有办法请问有没有其他下载软件推荐？
